### PR TITLE
Gamma distribution and updated list of distributions in readme

### PR DESCRIPTION
**Pull request by konimarti**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,7 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+*~
```
Comment (phillbaker) [inline .gitignore]: Better to put this in [your global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

File: README.rdoc [hunk 2]
```diff
@@ -37,12 +37,21 @@
...
+* Cauchy
 * Exponential
+* Gamma
+* Lognormal
+* Normal
+* Student t
+* Uniform
+* Weibull
+
+Classes for discrete distributions:
```
Comment (phillbaker) [inline README.rdoc:52]: 👍

**Review by phillbaker — APPROVED** [review]
Looks good - but let's drop the gitignore addition.

**Comment by konimarti** [comment]
Perfect. Thanks @phillbaker. I reset the .gitignore back to its original
state and updated pull request #15 accordingly.

**Comment by phillbaker** [comment]
Thanks!

**Pull request merged**